src/styles/vertical.scss

```diff
@@ -54,6 +54,10 @@ $tabitem-bias: 30px;
         padding-right: 4px;
         padding-top: 8px;
       }
+
+      .tab-item__icon {
+        margin-left: 0;
+      }
```

Comment:
Can this work on `tabs.scss` file, inestead of here?

Reply:
This margin-left is used so that the icons on the vertical sidebar don't go on the right when highlighted (because of the border), so this rule needs to apply only to the horizontal mode. Hence it has to stay in this file